ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v1/ConfigurationService.scala

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.index.v1
+
+import com.daml.ledger.participant.state.v1.Configuration
+
+import scala.concurrent.Future
+
+trait ConfigurationService {
+  def getLedgerConfiguration(): Future[Configuration]
```

Comment:
Wonder if we should already build this to support the streaming nature: https://github.com/digital-asset/daml/blob/master/ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/ledger_configuration_service.proto#L19

Reply:
yes, good point

Reply:
I changed it to a Stream